docs/resources/remote_docker_repository.md

```diff
@@ -36,11 +36,12 @@ The following arguments are supported, along with the [common list of arguments
 * `enable_token_authentication` - (Optional) Enable token (Bearer) based authentication.
 * `external_dependencies_enabled` - (Optional) Also known as 'Foreign Layers Caching' on the UI.
 * `external_dependencies_patterns` - (Optional) An allow list of Ant-style path patterns that determine which remote VCS roots Artifactory will
-  follow to download remote modules from, when presented with 'go-import' meta tags in the remote repository response. 
-  By default, this is set to '**' in the UI, which means that remote modules may be downloaded from any external VCS source.
+  follow to download remote modules from, when presented with 'go-import' meta tags in the remote repository response.
+  By default, this is set to '[**]' in the UI, which means that remote modules may be downloaded from any external VCS source.
   Due to SDKv2 limitations, we can't set the default value for the list.
-  This value must be assigned to the attribute manually, if user don't specify any other non-default values.
-  This attribute must be set together with `external_dependencies_enabled = true`.
+  This value '[**]' must be assigned to the attribute manually, if user don't specify any other non-default values.
+  We don't want to make this attribute required, but it must be set to avoid the state drift on update. Note: Artifactory assigns
```

Comment:
If it must be assigned, why don't we make it required?

Reply:
Making the attribute required can cause unwanted effects on the existing configurations.